spartanstay/spartanstay/src/main/java/com/spartanstay/spartanstay/controller/ListingsController.java

```diff
@@ -27,7 +27,7 @@ String getRooms(@RequestParam("destination") String destination, @RequestParam("
             destId = "1506246";
         }
 
-        return listingService.getListings(destId, checkIn.toString(), checkOut.toString(), "PRICE","1");
+        return listingService.getListings(destId, checkIn.toString(), checkOut.toString(), "PRICE","1", starRatings);
```

Comment:
If we can make this optional, we would want some conditional logic to make sure we're passing in something reasonable here.